keg_auth/core.py

```diff
@@ -243,10 +243,16 @@ def create_user_cli(self, email, extra_args):
 
     def create_user(self, user_kwargs, _commit=True):
         mail_enabled = user_kwargs.pop('mail_enabled', True)
+        permissions = user_kwargs.pop('permissions', [])
+        bundles = user_kwargs.pop('bundles', [])
+        groups = user_kwargs.pop('groups', [])
         from passlib.pwd import genword
         user_kwargs.setdefault('password', genword(entropy='secure'))
         user_class = self.entity_registry.user_cls
         user = user_class(**user_kwargs)
```

Comment:
I would think permissions/bundles/groups could just be allowed to pass in through user_kwargs.